linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -109,17 +118,36 @@ static Bom copyWithSnapshot(RepositorySystem repositorySystem, Bom bom)
   static Optional<String> findSnapshotVersion(
       RepositorySystem repositorySystem, RepositorySystemSession session, Artifact artifact)
       throws VersionRangeResolutionException {
+    String version = findHighestVersion(repositorySystem, session, artifact);
```

Comment:
Does this load from the local repo too? That's the only place snapshot versions are likely to be right now.